mindmeld/converter/dialogflow.py

```diff
@@ -225,12 +232,12 @@ def _create_intent_file(
                                 "Please create an entity for this.",
                                 df_meta[1:],
                             )
+                            entity_type = self.clean_name(mm_meta) + "_" + language
 
-                        entity_type = self.clean_name(mm_meta) + "_entries_" + language
                         part = "{" + df_text + "|" + entity_type + "}"
                     else:
                         entity_type = (
-                            self.clean_name(df_meta[1:]) + "_entries_" + language
+                            self.clean_name(df_meta[1:]) + "_" + language
```

Comment:
are you guaranteed that `language` is a string?

Reply:
Resolved this issue by making sure the function calling this function always has the language string set.